src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -0,0 +1,239 @@
+ */
+
+/*
+ * The following import statements makes available all the elements
+ *required by the components encompassed within
+ */
+
+import React, { lazy, Suspense, useContext, useEffect, useState } from "react";
+import { Menu } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { AnalysisOutputsProvider } from "../../../contexts/AnalysisOutputsContext";
+import { AnalysisSteps } from "./AnalysisSteps";
+import { PageWrapper } from "../../../components/page/PageWrapper";
+
+import { Link, Location, Router } from "@reach/router";
+
+import { Error } from "../../../components/icons/Error";
+import { Running } from "../../../components/icons/Running";
+import { Success } from "../../../components/icons/Success";
+import { SPACE_MD } from "../../../styles/spacing";
+import AnalysisError from "./AnalysisError";
+import { ContentLoading } from "../../../components/loader/ContentLoading";
+import { ANALYSIS } from "../routes";
+import { getNewickTree } from "../../../apis/analysis/analysis";
+
+const AnalysisBioHansel = React.lazy(() => import("./AnalysisBioHansel"));
+const AnalysisPhylogeneticTree = React.lazy(() =>
+  import("./AnalysisPhylogeneticTree")
+);
+
+const AnalysisSistr = React.lazy(() => import("./AnalysisSistr"));
+const AnalysisSettingsContainer = lazy(() =>
+  import("./settings/AnalysisSettingsContainer")
+);
+const AnalysisOutputFiles = lazy(() => import("./AnalysisOutputFiles"));
+const AnalysisProvenance = lazy(() => import("./AnalysisProvenance"));
+
+export default function Analysis() {
+  const { analysisContext } = useContext(AnalysisContext);
+  const DEFAULT_URL = `/analysis/${analysisContext.analysis.identifier}`;
+  const [treeDefault, setTreeDefault] = useState(null);
+
+  useEffect(() => {
+    if((analysisType === "PHYLOGENOMICS" || analysisType === "MLST_MENTALIST") && analysisContext.isCompleted) {
+      getNewickTree(analysisContext.analysis.identifier).then(data => {
+        if (data.newick !== null) {
+          setTreeDefault(true);
+        } else {
+          setTreeDefault(false);
+        }
+      });
+    }
+  }, []);
+
+  const title = (
+    <>
+      {analysisContext.analysisState === "COMPLETED" ? (
+        <Success />
+      ) : analysisContext.analysisState === "ERROR" ? (
+        <Error />
+      ) : (
+        <Running />
+      )}
+      {analysisContext.analysisName}
+    </>
+  );
+
+  const analysisType = analysisContext.analysisType;
+
+  const pathRegx = new RegExp(/\/analysis\/[0-9]+\/+([a-zA-Z_0-9]+)/);
+
+  const defaultKey = analysisContext.isCompleted
+    ? analysisType === "SISTR_TYPING"
+      ? ANALYSIS.SISTR
+      : analysisType === "BIO_HANSEL"
+      ? ANALYSIS.BIOHANSEL
+      : ((analysisType === "PHYLOGENOMICS" || analysisType === "MLST_MENTALIST") && treeDefault)
+      ? ANALYSIS.TREE
+      : ANALYSIS.OUTPUT
+    : analysisContext.isError
+    ? ANALYSIS.ERROR
+    : ANALYSIS.SETTINGS;
+
+  function getTabLinks() {
```

Comment:
Comment this please

Reply:
Updated in 5fab46ad6950f645c28784b4324f4c0a856e3df7